app/frontend/src/features/profile/edit/EditProfile.tsx

```diff
@@ -151,10 +153,30 @@ export default function EditProfileForm() {
     register("radius");
   }, [register]);
 
-  const onSubmit = handleSubmit((data) => {
-    resetUpdate();
-    updateUserProfile({ profileData: data, setMutationError: setErrorMessage });
-  });
+  const onSubmit = handleSubmit(
+    (data) => {
+      resetUpdate();
+      updateUserProfile(
+        {
+          profileData: data,
+          setMutationError: setErrorMessage,
+        },
+        {
+          // Scoll to top on submission error
+          onError: () => {
+            window.scroll({ top: 0 });
+          },
+        }
+      );
+    },
+    () => {
```

Comment:
```suggestion
    (errors) => {
```
It isn't working for me - I guess it's because `errors` from `useForm` isn't set until after this callback is executed? But the errors are also passed into the function so try this.